src/options.js

```diff
@@ -268,6 +268,16 @@ exports.bool = {
     singleGroups: false,
 
     /**
+     * When set to true, the use of VariableStatements are forbidden.
+     * For example:
+     *
+     *     // jshint varstmt: true
+     *
+     *     var a; // Warning: `var` declarations are forbidden. Use `let` or `const` instead.
+     */
+    varstmt: false,
```

Comment:
I think this name will make configuration files a little confusing since `varstmt: true` means "no `var` statements allowed". What about `novarstmt` or just `novar`?